components/desktop/gnome2/gnome-mount/Makefile

```diff
@@ -13,7 +13,7 @@
 # Copyright 2014 Alexander Pyhalov.  All rights reserved.
 #
 
-include ../../make-rules/shared-macros.mk
```

Comment:
Don't touch it, I'll  remove it in several days.

Reply:
OK. I can only fix it tomorrow evening.